internal/operators/manager.go

```diff
@@ -380,3 +380,21 @@ func (mgr *Manager) GetSupportedOperatorsByType(operatorType models.OperatorType
 
 	return operators
 }
+
+func EnsureLVMAndCNVNotEnabled(operators []*models.MonitoredOperator) error {
+	cnvEnabled := false
+	lvmEnabled := false
+	for _, updatedOperator := range operators {
+		if updatedOperator.Name == "lvm" {
+			lvmEnabled = true
+		}
+
+		if updatedOperator.Name == "cnv" {
+			cnvEnabled = true
+		}
+	}
+	if lvmEnabled && cnvEnabled {
```

Comment:
we are not allowed to install CNV & LVM on OCP 4.11 but we can do it on OCP 4.12